src/scenes/Engagement/Files/useUploadEngagementFiles.ts

```diff
@@ -15,13 +14,6 @@ import {
 
 type EngagementType = 'language' | 'internship';
 
-const isLanguageEngagementMutation = (
-  data:
-    | UploadInternshipEngagementGrowthPlanMutation
-    | UploadLanguageEngagementPnpMutation
-): data is UploadLanguageEngagementPnpMutation =>
-  'updateLanguageEngagement' in data;
-
 export const useUploadEngagementFile = (
```

Comment:
This hook should go away. Can't we just pass the mutation to the defined file card? If not we should have separate hooks dedicated to pnp & growthPlan.